src/brushes/pencil_brush.class.js

```diff
@@ -335,9 +335,25 @@
       } while (checkIndex < fullPath.length - 1)
       // always push the line ending
       newPath.push(fullPath[fullPath.length - 1])
-      return newPath
+      return newPath;
+    },
+
+    detectAndEnlargeSinglePointPath: function (path){
+      var lastIndex = path.length - 1;
+      for (var i = 0; i < lastIndex; i++){
+        if (path[i][1] !== path[i + 1][1] ||
+            path[i][2] !== path[i + 1][2]) {
+          // path contains at least one divergent point
+          return path;
```

Comment:
So as long as the path is not composed of all exactly the same point this returns. So it detects if a path IS a single point. 

Given the path is pruned early for dups I think all cases that this should happen would be where this is 1 (or two, dun recall the details of the pruning).

No comment here just thinking out loud :)